Dockerfile.worker

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+ENV CELERY_VERSION 4.0.2
+
+COPY workers/requirements.in .
+
+RUN pip install pip-tools && \
+    pip-compile requirements.in && \
+    pip install -r requirements.txt
+
+COPY workers/ .
+
+USER user
+
+CMD ["celery", "-A", "bioinformatics_mill_workers", "-c", "1", "worker"]
```

Comment:
Makes sense. I think I assumed from the file names "Docker.foreman" and "Docker.worker" that there was some Docker-based support for the pattern. A Google search for "Docker foreman" even turned up some meaty-looking hits...